adapters/multiline/multiline.go

```diff
@@ -59,6 +60,11 @@ func NewMultilineAdapter(route *router.Route) (a router.LogAdapter, err error) {
 	if pattern == "" {
 		pattern = `^\s`
 	}
+
+	separator := os.Getenv("MULTILINE_SEPARATOR")
+	if separator == "" {
+		separator = `\n`
```

Comment:
Use double quote instead of backtick (this will print out the literal `\n` instead of a new line).